core/platform/transactions/gae_transaction_services.py

```diff
@@ -20,15 +20,39 @@
 
 
 def run_in_transaction(fn, *args, **kwargs):
-    """Run a function in a transaction."""
+    """Run a function in a transaction.
+    Either all of the operations in the transaction are applied,
```

Comment:
Move to end of previous line; no need for line break at start of Either". If you want to start a new paragraph, have a blank line between the paragraphs.